wallets/leapextension.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "leapextension",
+  "pretty_name": "Leap Cosmos Wallet",
+  "website": "https://www.leapwallet.io/",
+  "git_repo": "",
+  "supported_chains": [
+    "cosmoshub",
+    "juno",
+    "osmosis",
+    "secretnetwork",
+    "agoric",
+    "akash",
+    "assetmantle",
+    "axelar",
+    "bitsong",
+    "cheqd",
+    "chihuahua",
+    "comdex",
+    "desmos",
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "irisnet",
+    "jackal",
+    "kujira",
+    "likecoin",
+    "mars",
```

Comment:
mars not registered to chain registry. Is it live yet?

Reply:
It is live on testnet. Let me know if this list should only include mainnet chains, I'll remove it from the list.

Reply:
Although we haven't decided yet on whether/how to include testnets herein, I feel quite confident that we likely won't allow chains that aren't yet registered to the chain registry--and neither 'mars' nor 'marstestnet' are registered yet. Thus, I'd like to ask that this be removed for now.

Reply:
Ok, I have removed mars from the list.